src/main/java/cpw/mods/modlauncher/serviceapi/ILaunchPluginService.java

```diff
@@ -193,6 +193,20 @@ default void initializeLaunch(ITransformerLoader transformerLoader,  NamedPath[]
     default void customAuditConsumer(String className, Consumer<String[]> auditDataAcceptor) {
     }
 
+    /**
+     * If this transformer should only run when a class' constant pool contains a given byte sequence,
+     * return it here. Multiple byte sequences will be treated as an OR relationship, and an empty
+     * array indicates that no filtering should be performed.
+     * <p></p>
+     * The return value of this method should be cached as it will be called frequently.
```

Comment:
A detail but you should really have `<p>` on this newline here instead of whatever that is :D